neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -50,6 +67,31 @@ def get_project(self, project_qualified_name):
         except HTTPNotFound:
             raise ProjectNotFound(project_qualified_name)
 
+    def _create_experiment_operations(self, entrypoint, params, tags) -> List[alpha_operation.Operation]:
+        """Returns operations required to initialize newly created experiment"""
+        init_operations = list()
+
+        # Assign source entrypoint
+        init_operations.append(alpha_operation.AssignString(
+            path=['source_code', 'entrypoint'],
+            value=entrypoint,
+        ))
+        # Assign experiment parameters
+        for p_name, p_val in params.items():
+            parameter_type, string_value = self._get_parameter_with_type(p_val)
+            operation_cls = alpha_operation.AssignFloat if parameter_type == 'double' else alpha_operation.AssignString
```

Comment:
In old client everything but numbers was converted to string, but in alpha we have `AssignDatetime` operation and series and sets which may be used as well.